python/src/TrendFactory_doc.i.in

```diff
@@ -60,7 +60,7 @@ Examples
 Define a scalar temporal Gaussian process on a mesh of dimension 1:
 
 >>> import openturns as ot
->>> myGrid = ot.RegularGrid(0.0, 1.0, 100)
+>>> myGrid = ot.RegularGrid(0.0, 0.1, 100)
```

Comment:
why this change ?

Reply:
You are right: No importance actually. I just wanted to make the grid coherent with the next graph which contains the Trend drawn in [0, 10]. 
I removed my modification.